src/main/java/Seat.java

```diff
@@ -0,0 +1,30 @@
+public class Seat {
+    int seatNumber;
+    int rowNumber;
+    boolean isTaken;
+
+
+
+  static String seatProperties(int seatNumber, int rowNumber, boolean isTaken) {
```

Comment:
Zbędna metoda. Od tego masz metodę toString():
https://www.javatpoint.com/understanding-toString()-method#:~:text=The%20toString()%20method%20returns,depends%20on%20your%20implementation.